core/players-handbook-2024/classes/class-monk.xml

```diff
@@ -0,0 +1,531 @@
+		<description>
+			<p>Whenever you deal damage with your Unarmed Strike, it can deal your choice of Force damage or its normal damage type.</p>
+		</description>
+		<sheet alt="Empowered Strikes">
+			<description>You can choose to deal Force damage for your Unarmed Strikes.</description>
+		</sheet>
+	</element>
+
+
+	<element name="Level 7: Evasion" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_MONK_EVASION">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_EVASION</requirements>
+		<description>
+			<p>When you're subjected to an effect that allows you to make a Dexterity saving throw to take only half damage, you instead take no damage if you succeed on the saving throw and only half damage if you fail.</p>
+			<p class="indent">You don't benefit from this feature if you have the Incapacitated condition.</p>
+		</description>
+		<sheet alt="Evasion">
+			<description>When you are subjected to an effect that allows you to make a Dexterity saving throw to take only half damage and you do not have the Incapacitated condition, you instead take no damage if you succeed on the saving throw, and only half damage if you fail.</description>
+		</sheet>
+	</element>
+
+
+	<element name="Level 9: Acrobatic Movement" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_MONK_ACROBATIC_MOVEMENT">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_ACROBATIC_MOVEMENT</requirements>
+		<description>
+			<p>While you aren't wearing armor or wielding a Shield, you gain the ability to move along vertical surfaces and across liquids on your turn without falling during the movement.</p>
+		</description>
+		<sheet alt="Acrobatic Movement">
+			<description>You can move along vertical surfaces and across liquids on your turn without falling while you are not wearing armor or wielding a shield.</description>
+		</sheet>
+	</element>
+
+
+	<element name="Level 10: Heightened Focus" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_MONK_HEIGHTENED_FOCUS">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_MONK_HEIGHTENED_FOCUS</requirements>
+		<description>
+			<p>Your Flurry of Blows, Patient Defense, and Step of the Wind gain the following benefits.</p>
+			<p><em><strong>Flurry of Blows.</strong></em> You can expend 1 Focus Point to use Flurry of Blows and make three Unarmed Strikes with it instead of two.</p>
+			<p><em><strong>Patient Defense.</strong></em> When you expend a Focus Point to use Patient Defense, you gain a number of Temporary Hit Points equal to two rolls of your Martial Arts die.</p>
+			<p><em><strong>Step of the Wind.</strong></em> When you expend a Focus Point to use Step of the Wind, you can choose a willing creature within 5 feet of yourself that is Large or smaller. You move the creature with you until the end of your turn. The creature's movement doesn't provoke Opportunity Attacks.</p>
```

Comment:
```suggestion
			<p class="indent"><em><strong>Flurry of Blows.</strong></em> You can expend 1 Focus Point to use Flurry of Blows and make three Unarmed Strikes with it instead of two.</p>
			<p class="indent"><em><strong>Patient Defense.</strong></em> When you expend a Focus Point to use Patient Defense, you gain a number of Temporary Hit Points equal to two rolls of your Martial Arts die.</p>
			<p class="indent"><em><strong>Step of the Wind.</strong></em> When you expend a Focus Point to use Step of the Wind, you can choose a willing creature within 5 feet of yourself that is Large or smaller. You move the creature with you until the end of your turn. The creature's movement doesn't provoke Opportunity Attacks.</p>
```